### PR TITLE
GROK-15726: Help: Diff Studio fitting & solver control

### PR DESCRIPTION
**Pull request by vmakarichev**
Added the fitting feature docs & solver control section

### CODE REVIEW

File: help/compute/diff-studio.md [hunk 1]
```diff
@@ -29,30 +29,53 @@ To run **Diff Studio**:
...
+  * [Monte Carlo](compute.md#monte-carlo)
+  * [Sobol](compute.md#sobol)
+  * [Grid](compute.md#grid)
 * Analyze model evaluations
 
 ![Run Sens Analysis](pics/DiffStudio-run-sens-analysis.gif)
 
+### Parameter Optimization
+
+Find input conditions leading to specified output constraints using the [Parameter Optimization](compute.md#input-parameter-optimization) feature. It computes inputs minimizing deviation measured by [loss function](https://en.wikipedia.org/wiki/Loss_function). Run it directly from Diff Studio:
```
[the remark this answers is not in the document]
Comment (skalkin) [reply help/compute/diff-studio.md:58]: => It finds input values that minimize deviation measured by...

File: help/compute/diff-studio.md [hunk 2]
```diff
@@ -353,7 +376,42 @@ The export feature provides an extension of your project with [scripting](script
 * non-elementary and special functions' use
 * Datagrok packages' functions call
 
+## Solver management
```
Comment (skalkin) [inline help/compute/diff-studio.md:379]: Probably "Solver tuning" or "Solver settings" would be better?

File: help/compute/diff-studio.md [hunk 3]
```diff
@@ -353,7 +376,42 @@ The export feature provides an extension of your project with [scripting](script
...
+
+Diff Studio implements the following [Rosenbrock–Wanner](https://doi.org/10.1016/j.cam.2015.03.010) methods for solving ODEs:
+
+|Method|Value|
+|-------------|--------|
+|The modified Rosenbrock triple|`'mrt'`|
+|The ROS3PRw method|`'ros3prw'`|
+|The ROS34PRw method|`'ros34prw'`|
+
+By default, Diff Studio uses ROS34PRw. You may specify the method as follows:
```
Comment (skalkin) [inline help/compute/diff-studio.md:395]: Too long to read. Let's make one example with three options, instead.

**Review by skalkin — APPROVED** [review]
Looks great, but let's also address my comments

File: help/compute/diff-studio.md [hunk 4]
```diff
@@ -29,30 +29,53 @@ To run **Diff Studio**:
...
-* Click **F5** or go to **Run** tab to re-run calculations and see updated data.
+* Press **F5** or go to **Run** tab to re-run calculations and see updated data.
 
 ![Run Diff Studio](pics/DiffStudio-run.gif)
 
 ### Loading and saving data
 
-* **To save formulas** in a local file, press the <i class="fas fa-save"></i> **Save** icon on the ribbon, and find the *ivp*-file in Downloads. You can open and edit this file using any text editor.
-* **To load formulas** from a local file, press the <i class="fas fa-folder-open"></i> **Load...** on the ribbon, choose **From file...**
+* **To save formulas** in a local file, click the <i class="fas fa-save"></i> **Save** icon on the ribbon, and find the *ivp*-file in Downloads. You can open and edit this file using any text editor.
```
[the remark this answers is not in the document]
Comment (vmakarichev) [reply help/compute/diff-studio.md:38]: Fixed

File: help/compute/diff-studio.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (vmakarichev) [reply help/compute/diff-studio.md:58]: Done

File: help/compute/diff-studio.md (hunk 2 above)
Comment (vmakarichev) [reply help/compute/diff-studio.md:379]: Replaced by "Solver settings"

File: help/compute/diff-studio.md (hunk 3 above)
Comment (vmakarichev) [reply help/compute/diff-studio.md:395]: Reduced

**Pull request merged**